jwst/associations/lib/rules_level3.py

```diff
@@ -313,18 +313,54 @@ def __init__(self, *args, **kwargs):
 
     @property
     def dms_product_name(self):
-        """Define product name."""
-        target = self._get_target()
+        return dms_product_name_noopt(self)
 
-        instrument = self._get_instrument()
 
-        product_name = 'jw{}-{}_{}_{}'.format(
-            self.data['program'],
-            self.acid.id,
-            target,
-            instrument
-        )
-        return product_name.lower()
+@RegistryMarker.rule
+class Asn_Lv3MIRMRSAux(AsnMixin_AuxData, AsnMixin_Spectrum):
```

Comment:
If this essentially creates the same kind of ASN's as `Lv3NRSIFUBackground` should we perhaps name it `Lv3MIRMRSBackground` for consistency?